lambda-function-sqs-report-batch-item-failures/example.php

```diff
@@ -0,0 +1,29 @@
+//Make sure you have Bref installed and the necessary dependencies. Run the following commands in your project directory:
+//composer require bref/bref
+//composer install
+
+
+<?php
+
+use Bref\Context\Context;
+use Bref\Event\SqsEvent;
+
+
+function lambda_handler(SQSEvent $event, Context $context) {
```

Comment:
Based on the [Bref documentation](https://bref.sh/docs/runtimes/function#php-classes) You need to either return a function or a class for the handler. I suggest rewriting this handler into a class as it's more aligned to the other code samples for PHP.